src/Components.tsx

```diff
@@ -13,19 +13,19 @@ export const ConnectPayouts = (): JSX.Element => {
 };
 
 export const ConnectPaymentDetails = ({
-  chargeId,
+  payment,
```

Comment:
How do we make sure this is backwards compatible? Or is this going out with the major version bump?

Reply:
I was thinking about this too: to make this backward compatible, I think we need to make `payment` optional or make both of them optional, and both ways seem a little strange.

Do we anticipate a major version in the near future? If so I think it makes sense to go out with that.

Reply:
Yup we'll def do a major change with GA since we're also changing the initialization pattern and adding beta react components to GA. But I'm not sure if we'll need to do a minor publish before then, that will have to include this pr

Reply:
> But I'm not sure if we'll need to do a minor publish before then, that will have to include this pr

Then I think we can make this backward compatible first, I'll create a new branch that make this only accept `payment`, and we can merge that branch before bumping major version. Does this sound good to you?

Reply:
@jojo-stripe I think this feels to me like a minor version bump (not a patch version bump), and your change that changes the initialization can be a major version bump. I don't think we have to aim to keep this package fully backwards compatible indefinitely.

Reply:
I'm peaceful with that, most users don't autoupdate with minor changes right, only patch